core/src/util/inflateBuffer.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include <vector>
+#include <string.h>
+
+namespace Tangram {
+
+int inflate(const char* _data, size_t _size, std::vector<char>& dst);
```

Comment:
This is awesome! Perhaps we should have a deflate function as well for when we populate the cache? As I was developing, I noticed that the cache got large pretty quickly.